app/javascript/components/AudioPlayer.tsx

```diff
@@ -98,7 +98,7 @@ export const AudioPlayer = (props: Props) => {
       />
       {isLoaded ? (
         <>
-          <div role="toolbar">
+          <div role="toolbar" className="flex items-center gap-2 text-xl leading-[1.3]">
```

Comment:
Does the `leading` make much difference? (especially as this appears to just be icons)

I'd rather not add it (or use one of Tailwind's default named leadings) unless it looks bad